src/plugins/intel_cpu/src/nodes/executors/acl/acl_eltwise.cpp

```diff
@@ -10,6 +10,8 @@ namespace intel_cpu {
 
 using namespace arm_compute;
 
+static std::mutex mtx_ifunc;
```

Comment:
if it's really required, you need to replace static variable with a function:
```c++
static A & get_a() {
    static A a;
    return a;
}
```
to prevent issues with global variables initialization.